fix(components) : [checkbox-group] fix Wrong type checking

ytx222 opened the pull request:
Every time you choose, there will be a warning on the issue, but you should have asked for the issue, but by the way, the code has been found   

==================  
每次选择之后都会有警告信息,本来打算提issue的,不过顺便看了下代码,找到原因了
[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG5jb25zdCBhYWEgPSByZWYoW10pXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8cD5cbiAgICA8ZWwtaWNvbiBjb2xvcj1cInZhcigtLWVsLWNvbG9yLXByaW1hcnkpXCI+PEVsZW1lbnRQbHVzIC8+PC9lbC1pY29uPlxuICAgIEVsZW1lbnQgUGx1cyB7eyBFcFZlcnNpb24gfX0gKyBWdWUge3sgdnVlVmVyc2lvbiB9fVxuICA8L3A+XG4gIDxlbC1jaGVja2JveC1ncm91cCB2LW1vZGVsPVwiYWFhXCI+XG4gICAgPGVsLWNoZWNrYm94XG4gICAgICB2LWZvcj1cIihpdGVtMiwgaTIpIGluIFsxLCAyLCAzLCA0LCA1XVwiXG4gICAgICA6a2V5PVwiaXRlbTJcIlxuICAgICAgOmxhYmVsPVwiaXRlbTJcIlxuICAgID5cbiAgICA8L2VsLWNoZWNrYm94PlxuICA8L2VsLWNoZWNrYm94LWdyb3VwPlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydF9tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwiX28iOnt9fQ==)

fix #8125

tolking commented:
@ytx222 似乎你的提交信息不正确

应该是 `fix(components): [checkbox-group]` 而不是 `fix(components) : [el-checkbox-group]`

你或许应该修改 git 历史提交信息

ytx222 commented:
我的最后一次提交信息是正确的,之前的提交信息是手动输入的,最后一次是使用npm run cz来生成的,在这种情况下,合并消息是不是以最后一次的提交信息为准的

tolking commented:
> 你的所有 commit(s) 会被合并为一个 commit 来被验证，所以你可能需要修改你之前的 commit(s) 消息。
如果你不知道如何来修改之前已经提交的记录，请参考[让你的 Git 提交历史保持干净](https://about.gitlab.com/blog/2018/06/07/keeping-git-commit-history-clean/)来修改。

机器人会检查所有信息。相关修改操作可以去搜索

ytx222 commented:
好的,我研究下如何清除之前的提交记录

ytx222 commented:
我已经进行了修改,这次应该正确了,谢谢

tolking commented on packages/components/checkbox/src/checkbox.ts:
```diff
@@ -303,13 +309,20 @@ const useEvent = (
   }
 }
 
+type checkboxValueType = string | number | boolean
```
定义类型通常以大写字母开头

ytx222 replied:
好的,需要我这边修改吗?

tolking replied:
需要

ytx222 committed:
fix(components): [el-checkbox-group] fix Wrong type checking

el-checkbox-group uses the emit type of el-checkbox,
resulting in a type check error in the console during development

tolking commented:
感谢你的贡献!
